humanguide/layout/01-app-main-view.rst

```diff
@@ -0,0 +1,35 @@
+App main view
+=============
+
+When an app starts, shows a generic splash screen (as in the image) or a
+custom one, then the main view.
+
+.. figure:: /_static/images/humanguide/Splash-Main.png
+   :alt: Splash-Main
+
+   Splash-Main
+
+The standard parts of an main view or any other page may be:
+
+Header
+------
+
+Display app name, leading area and trailing area icons.
+
+.. figure:: /_static/images/humanguide/21.png
+
+In this case, showing title and trailing actions
+
+Tabbed content
+--------------
+
+To organize information under tabs
+
+.. figure:: /_static/images/humanguide/22.png
+
+Bottom edge menu
+----------------
+
+For quick to common actions
```

Comment:
This should link to the page about "common quick actions". Do we want to keep that wording, though? It's not very clear. Maybe call it a "Quick action" instead? Or a "Creation action"? I guess the bottom edge doesn't always create something though.